src/parse-index.js

```
@@ -251,14 +251,57 @@ module.exports = {
+                try {
+                  // Safely extract text from tokens with proper error handling
+                  return [
+                    columnName,
+                    tokens[0].items.map(item => {
+                      try {
+                        // Check if the token structure is as expected
+                        if (item &&
+                            item.tokens &&
+                            item.tokens[0] &&
+                            item.tokens[0].tokens &&
+                            item.tokens[0].tokens[0] &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)